webapi/src/main/scala/org/knora/webapi/messages/util/search/SparqlTransformer.scala

```diff
@@ -97,7 +114,7 @@ object SparqlTransformer {
 
     entityStr
       .replaceAll("[:/.#-]", "")
-      .replaceAll("\\s", "") // TODO: check if this is complete and if it could lead to collision of variable names
+      .replaceAll("\\s", "") // TODO-old: check if this is complete and if it could lead to collision of variable names
```

Comment:
Why TODO-old? ;) I would just leave it as TODO.

Reply:
done